RecoTracker/LSTCore/src/alpaka/Kernels.h

```diff
@@ -0,0 +1,424 @@
+#ifndef Kernels_cuh
+#define Kernels_cuh
+
+#include "RecoTracker/LSTCore/interface/alpaka/Constants.h"
+#include "RecoTracker/LSTCore/interface/alpaka/Module.h"
+
+#include "Hit.h"
+#include "MiniDoublet.h"
+#include "Segment.h"
+#include "Triplet.h"
+#include "Quintuplet.h"
+#include "PixelTriplet.h"
+
+namespace SDL {
+  ALPAKA_FN_ACC ALPAKA_FN_INLINE void rmQuintupletFromMemory(struct SDL::quintuplets& quintupletsInGPU,
```

Comment:
I have a feeling these non-templated functions should be placed in `ALPAKA_ACCELERATOR_NAMESPACE`.